drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -335,11 +331,14 @@ function DragAndDropTemplates(configuration) {
         );
     };
 
-    var getFixedSizingFeedbackDetailTemplate = function(ctx) {
-        var attempts_used_html = h('div.attempts', [
-            gettext('{used} / {total}').replace('{used}', ctx.attempts).replace('{total}', ctx.max_attempts),
-            h('small', gettext('Attempts'))
-        ]);
+    var getAssessmentModeFixedSizingFeedbackDetailTemplate = function(ctx) {
+        var attempts_used_html = null;
+        if (ctx.max_attempts) {
+            attempts_used_html = h('div.attempts', [
+                gettext('{used} / {total}').replace('{used}', ctx.attempts).replace('{total}', ctx.max_attempts),
+                h('small', gettext('Attempts'))
+            ]);
+        }
```

Comment:
Instead of completely disabling the display, perhaps it could be something like this:
```javascript
let message = ctx.max_attempts 
    ? gettext('{used} / {total}').replace('{used}', ctx.attempts).replace('{total}', ctx.max_attempts)
    : ctx.attempts.toString();
attempts_used_html = h('div.attempts', [
    message,
    h('small', gettext('Attempts'))
]);
```
This will show the message as "8 / 12 Attempts"  if there is a ``max_attempts``, and "8 Attempts" otherwise.